python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -544,6 +557,10 @@ def measureCrosstalkCoefficients(self, ratios, rejIter, rejSigma):
         ----------
         ratios : `dict` [`dict` [`numpy.ndarray`]]
            Catalog of arrays of ratios.  The ratio arrays are one-dimensional
+        ordering : `list` [`str`]
+           List to use as a mapping between amplifier names (the
+           elements of the list) and their position in the output
+           calibration (the matching index of the list).
```

Comment:
This may be None, I guess?  If so, please say that's okay and what happens if it is None.